packages/frontend/src/navbar.ts

```diff
@@ -177,6 +177,11 @@ export const navbarItemDef = reactive({
 		show: computed(() => $i != null),
 		to: `/@${$i?.username}`,
 	},
+	support: {
+		title: i18n.ts.misskeyIOSupport,
```

Comment:
```suggestion
		title: i18n.tsx.supportThisInstance({ name: instance.name ?? host }),
```
既にlocaleに同じ目的のものがあるので追加しなくていいと思う